### ISSUE TITLE
Session Fixation False positive

### ISSUE THREAD

**Issue by zapbot** [zaproxy/zaproxy#1142]
```
What steps will reproduce the problem?

1.  Prepare a Web application that has the following characteristics:

- You log in to the application by accessing to a Login URL via POST method.
- It provides a session cookie when accessing the Login URL without session cookies
via GET method.(in this case, you cannot log in successfully)
* e.g. web applications of CakePHP

2. Log in to the application manually.

3. Register the url as a Login URL

4. Access the Login URL via GET method manually.

5. Do an active scan against the Login URL with GET method on Site tab to check Session
Fixation.

What is the expected output? What do you see instead?
There seems to be no vulnerabilities, but I got a Session Fixation alert in Alert tab.

What version of the product are you using? On what operating system?
ZAP v2.2.2

Please provide any additional information below.
I guess it's better to check if the target url is a Login URL by the HTTP method used.
```

Original issue reported on code.google.com by `yukisov` on 2014-04-09 10:25:12

**Comment by kingthorin** [comment]
Something like: https://stackoverflow.com/questions/19531044/creating-a-very-simple-1-username-password-login-in-php might provide content by which to test this FP (versus trying to come up with a version of CakePHP and old docs/guidance from 2014/2015)...

**Comment by kingthorin** [comment]
I've been able to recreate this issue leveraging one of the examples from stackoverflow.

**Comment by thc202** [comment]
Fixed in zaproxy/zap-extensions#1491.

**Comment by thc202** [comment]
Released in version 24 of Active scanner rules (beta) add-on.
